### ISSUE TITLE
Warning: Class scanner encountered an error while scanning class

### ISSUE THREAD

**Issue by ratchanonp** [SBPrime/AsyncWorldEdit#392]
CONSOLE
==========
```console
[20:08:23 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[20:08:23 INFO]: [AsyncWorldEdit] Warning: Class scanner encountered an error while scanning class
[20:08:23 INFO]: [AsyncWorldEdit] Errors:
[20:08:23 INFO]: [AsyncWorldEdit]  - Unpack field 'serialVersionUID': java.lang.reflect.InaccessibleObjectException, Unable to make field static final long java.util.HashSet.serialVersionUID accessible: module java.base does not "opens java.util" to unnamed module @7fee423
[20:08:23 INFO]: [AsyncWorldEdit]  - Unpack field 'map': java.lang.reflect.InaccessibleObjectException, Unable to make field private transient java.util.HashMap java.util.HashSet.map accessible: module java.base does not "opens java.util" to unnamed module @7fee423
[20:08:23 INFO]: [AsyncWorldEdit]  - Unpack field 'PRESENT': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final java.lang.Object java.util.HashSet.PRESENT accessible: module java.base does not "opens java.util" to unnamed module @7fee423
[20:08:23 INFO]: [AsyncWorldEdit] Class: class java.util.HashSet
[20:08:23 INFO]: [AsyncWorldEdit] Object: []
[20:08:23 INFO]: [AsyncWorldEdit] Send this message to the author of the plugin!
[20:08:23 INFO]: [AsyncWorldEdit] https://github.com/SBPrime/AsyncWorldEdit/issues
[20:08:23 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
```

***More Log*** : [https://pastebin.com/6Hk0R11r](https://pastebin.com/6Hk0R11r)

INFO
==========
- ***OS***: Ubuntu 18.04.5 
- ***Java***: 
    - openjdk 16.0.1 2021-04-20
    - OpenJDK Runtime Environment (build 16.0.1+9-24)
    - OpenJDK 64-Bit Server VM (build 16.0.1+9-24, mixed mode, sharing) 
- ***Server*** :  PaperMC 1.17 Build 30
- ***WorldEdit***: WorldEdit-bukkit 7.3.0 Build 5835

**Comment by FreedomForeverMC** [comment]
I'm having the same issue, the server crashed aswell

**Comment by cubanrebelde** [comment]
Having the same Problem, Plugins running: [22:20:03 INFO]: Plugins (26): AngelChest, AsyncWorldEdit, BlockLocker, BottledExp, ChestSort, CMILib, CoreProtect, dynmap*, Essentials, EssentialsChat, EssentialsGeoIP, EssentialsProtect, EssentialsSpawn, FarmControl, Jobs, LuckPerms, mcMMO, ProtocolLib, QuickShop, SilkSpawners, Vault, VillagerOptimiser, WorldBorder, WorldEdit, WorldGuard, WorldGuardExtraFlags
>


[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] Warning: Class scanner encountered an error while scanning class
[22:15:30 INFO]: [AsyncWorldEdit] Errors:
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'serialVersionUID': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final long java.util.ArrayList.serialVersionUID accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'DEFAULT_CAPACITY': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final int java.util.ArrayList.DEFAULT_CAPACITY accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'EMPTY_ELEMENTDATA': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final java.lang.Object[] java.util.ArrayList.EMPTY_ELEMENTDATA accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'DEFAULTCAPACITY_EMPTY_ELEMENTDATA': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final java.lang.Object[] java.util.ArrayList.DEFAULTCAPACITY_EMPTY_ELEMENTDATA accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'elementData': java.lang.reflect.InaccessibleObjectException, Unable to make field transient java.lang.Object[] java.util.ArrayList.elementData accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'size': java.lang.reflect.InaccessibleObjectException, Unable to make field private int java.util.ArrayList.size accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'modCount': java.lang.reflect.InaccessibleObjectException, Unable to make field protected transient int java.util.AbstractList.modCount accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit] Class: class java.util.ArrayList
[22:15:30 INFO]: [AsyncWorldEdit] Object: [(1, 0, 0), (-1, 0, 0), (0, 0, 1), (0, 0, -1), (0, -1, 0)]
[22:15:30 INFO]: [AsyncWorldEdit] Send this message to the author of the plugin!
[22:15:30 INFO]: [AsyncWorldEdit] https://github.com/SBPrime/AsyncWorldEdit/issues
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] Warning: Class scanner encountered an error while scanning class
[22:15:30 INFO]: [AsyncWorldEdit] Errors:
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'elements': java.lang.reflect.InaccessibleObjectException, Unable to make field transient java.lang.Object[] java.util.ArrayDeque.elements accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'head': java.lang.reflect.InaccessibleObjectException, Unable to make field transient int java.util.ArrayDeque.head accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'tail': java.lang.reflect.InaccessibleObjectException, Unable to make field transient int java.util.ArrayDeque.tail accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'MAX_ARRAY_SIZE': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final int java.util.ArrayDeque.MAX_ARRAY_SIZE accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'serialVersionUID': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final long java.util.ArrayDeque.serialVersionUID accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit] Class: class java.util.ArrayDeque
[22:15:30 INFO]: [AsyncWorldEdit] Object: [(913, 71, -544)]
[22:15:30 INFO]: [AsyncWorldEdit] Send this message to the author of the plugin!
[22:15:30 INFO]: [AsyncWorldEdit] https://github.com/SBPrime/AsyncWorldEdit/issues
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] Warning: Class scanner encountered an error while scanning class
[22:15:30 INFO]: [AsyncWorldEdit] Errors:
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'serialVersionUID': java.lang.reflect.InaccessibleObjectException, Unable to make field static final long java.util.HashSet.serialVersionUID accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'map': java.lang.reflect.InaccessibleObjectException, Unable to make field private transient java.util.HashMap java.util.HashSet.map accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'PRESENT': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final java.lang.Object java.util.HashSet.PRESENT accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit] Class: class java.util.HashSet
[22:15:30 INFO]: [AsyncWorldEdit] Object: [(913, 71, -544)]
[22:15:30 INFO]: [AsyncWorldEdit] Send this message to the author of the plugin!
[22:15:30 INFO]: [AsyncWorldEdit] https://github.com/SBPrime/AsyncWorldEdit/issues
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] Warning: Class scanner encountered an error while scanning class
[22:15:30 INFO]: [AsyncWorldEdit] Errors:
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'serialVersionUID': java.lang.reflect.InaccessibleObjectException, Unable to make field static final long java.util.HashSet.serialVersionUID accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'map': java.lang.reflect.InaccessibleObjectException, Unable to make field private transient java.util.HashMap java.util.HashSet.map accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'PRESENT': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final java.lang.Object java.util.HashSet.PRESENT accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit] Class: class java.util.HashSet
[22:15:30 INFO]: [AsyncWorldEdit] Object: [com.sk89q.worldedit.function.mask.RegionMask@b782d2a, com.sk89q.worldedit.function.mask.MaskUnion@43238db7, com.sk89q.worldedit.function.mask.BoundedHeightMask@49600457]
[22:15:30 INFO]: [AsyncWorldEdit] Send this message to the author of the plugin!
[22:15:30 INFO]: [AsyncWorldEdit] https://github.com/SBPrime/AsyncWorldEdit/issues
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] Warning: Class scanner encountered an error while scanning class
[22:15:30 INFO]: [AsyncWorldEdit] Errors:
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'serialVersionUID': java.lang.reflect.InaccessibleObjectException, Unable to make field static final long java.util.HashSet.serialVersionUID accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'map': java.lang.reflect.InaccessibleObjectException, Unable to make field private transient java.util.HashMap java.util.HashSet.map accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'PRESENT': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final java.lang.Object java.util.HashSet.PRESENT accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit] Class: class java.util.HashSet
[22:15:30 INFO]: [AsyncWorldEdit] Object: [com.sk89q.worldedit.function.mask.BlockTypeMask@610532c0, com.sk89q.worldedit.function.mask.Masks$1@364dc82a]
[22:15:30 INFO]: [AsyncWorldEdit] Send this message to the author of the plugin!
[22:15:30 INFO]: [AsyncWorldEdit] https://github.com/SBPrime/AsyncWorldEdit/issues
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] Warning: Class scanner encountered an error while scanning class
[22:15:30 INFO]: [AsyncWorldEdit] Errors:
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'serialVersionUID': java.lang.reflect.InaccessibleObjectException, Unable to make field static final long java.util.HashSet.serialVersionUID accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'map': java.lang.reflect.InaccessibleObjectException, Unable to make field private transient java.util.HashMap java.util.HashSet.map accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'PRESENT': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final java.lang.Object java.util.HashSet.PRESENT accessible: module java.base does not "opens java.util" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit] Class: class java.util.HashSet
[22:15:30 INFO]: [AsyncWorldEdit] Object: [minecraft:lava]
[22:15:30 INFO]: [AsyncWorldEdit] Send this message to the author of the plugin!
[22:15:30 INFO]: [AsyncWorldEdit] https://github.com/SBPrime/AsyncWorldEdit/issues
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] Warning: Class scanner encountered an error while scanning class
[22:15:30 INFO]: [AsyncWorldEdit] Errors:
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'serialVersionUID': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final long java.util.concurrent.locks.ReentrantLock.serialVersionUID accessible: module java.base does not "opens java.util.concurrent.locks" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'sync': java.lang.reflect.InaccessibleObjectException, Unable to make field private final java.util.concurrent.locks.ReentrantLock$Sync java.util.concurrent.locks.ReentrantLock.sync accessible: module java.base does not "opens java.util.concurrent.locks" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit] Class: class java.util.concurrent.locks.ReentrantLock
[22:15:30 INFO]: [AsyncWorldEdit] Object: java.util.concurrent.locks.ReentrantLock@2551a4f2[Unlocked]
[22:15:30 INFO]: [AsyncWorldEdit] Send this message to the author of the plugin!
[22:15:30 INFO]: [AsyncWorldEdit] https://github.com/SBPrime/AsyncWorldEdit/issues
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] Warning: Class scanner encountered an error while scanning class
[22:15:30 INFO]: [AsyncWorldEdit] Errors:
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'serialVersionUID': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final long java.util.concurrent.locks.ReentrantLock.serialVersionUID accessible: module java.base does not "opens java.util.concurrent.locks" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'sync': java.lang.reflect.InaccessibleObjectException, Unable to make field private final java.util.concurrent.locks.ReentrantLock$Sync java.util.concurrent.locks.ReentrantLock.sync accessible: module java.base does not "opens java.util.concurrent.locks" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit] Class: class java.util.concurrent.locks.ReentrantLock
[22:15:30 INFO]: [AsyncWorldEdit] Object: java.util.concurrent.locks.ReentrantLock@7349473[Unlocked]
[22:15:30 INFO]: [AsyncWorldEdit] Send this message to the author of the plugin!
[22:15:30 INFO]: [AsyncWorldEdit] https://github.com/SBPrime/AsyncWorldEdit/issues
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] Warning: Class scanner encountered an error while scanning class
[22:15:30 INFO]: [AsyncWorldEdit] Errors:
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'serialVersionUID': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final long java.util.concurrent.locks.ReentrantLock.serialVersionUID accessible: module java.base does not "opens java.util.concurrent.locks" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'sync': java.lang.reflect.InaccessibleObjectException, Unable to make field private final java.util.concurrent.locks.ReentrantLock$Sync java.util.concurrent.locks.ReentrantLock.sync accessible: module java.base does not "opens java.util.concurrent.locks" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit] Class: class java.util.concurrent.locks.ReentrantLock
[22:15:30 INFO]: [AsyncWorldEdit] Object: java.util.concurrent.locks.ReentrantLock@26c03430[Unlocked]
[22:15:30 INFO]: [AsyncWorldEdit] Send this message to the author of the plugin!
[22:15:30 INFO]: [AsyncWorldEdit] https://github.com/SBPrime/AsyncWorldEdit/issues
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] Warning: Class scanner encountered an error while scanning class
[22:15:30 INFO]: [AsyncWorldEdit] Errors:
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'serialVersionUID': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final long java.util.concurrent.locks.ReentrantLock.serialVersionUID accessible: module java.base does not "opens java.util.concurrent.locks" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'sync': java.lang.reflect.InaccessibleObjectException, Unable to make field private final java.util.concurrent.locks.ReentrantLock$Sync java.util.concurrent.locks.ReentrantLock.sync accessible: module java.base does not "opens java.util.concurrent.locks" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit] Class: class java.util.concurrent.locks.ReentrantLock
[22:15:30 INFO]: [AsyncWorldEdit] Object: java.util.concurrent.locks.ReentrantLock@19066a18[Unlocked]
[22:15:30 INFO]: [AsyncWorldEdit] Send this message to the author of the plugin!
[22:15:30 INFO]: [AsyncWorldEdit] https://github.com/SBPrime/AsyncWorldEdit/issues
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:30 INFO]: [AsyncWorldEdit] Warning: Class scanner encountered an error while scanning class
[22:15:30 INFO]: [AsyncWorldEdit] Errors:
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'serialVersionUID': java.lang.reflect.InaccessibleObjectException, Unable to make field private static final long java.util.concurrent.locks.ReentrantLock.serialVersionUID accessible: module java.base does not "opens java.util.concurrent.locks" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit]  - Unpack field 'sync': java.lang.reflect.InaccessibleObjectException, Unable to make field private final java.util.concurrent.locks.ReentrantLock$Sync java.util.concurrent.locks.ReentrantLock.sync accessible: module java.base does not "opens java.util.concurrent.locks" to unnamed module @2fd0319d
[22:15:30 INFO]: [AsyncWorldEdit] Class: class java.util.concurrent.locks.ReentrantLock
[22:15:30 INFO]: [AsyncWorldEdit] Object: java.util.concurrent.locks.ReentrantLock@5040cafd[Unlocked]
[22:15:30 INFO]: [AsyncWorldEdit] Send this message to the author of the plugin!
[22:15:30 INFO]: [AsyncWorldEdit] https://github.com/SBPrime/AsyncWorldEdit/issues
[22:15:30 INFO]: [AsyncWorldEdit] -----------------------------------------------------------------------
[22:15:36 INFO]: Cubanrevel issued server command: //undo
[22:21:07 INFO]: [AsyncWorldEdit] Undo cleanup started...
[22:21:07 INFO]: [AsyncWorldEdit]        * plugins\AsyncWorldEdit\undo\f654a811-f474-4368-ba8e-c0fe8e600403\ts.17a322ec580.3bdfb628.0...ok
[22:21:07 INFO]: [AsyncWorldEdit]        * plugins\AsyncWorldEdit\undo\f654a811-f474-4368-ba8e-c0fe8e600403\ts.17a322ec580.3bdfb628.0.idx...ok
[22:21:07 INFO]: [AsyncWorldEdit]        * plugins\AsyncWorldEdit\undo\f654a811-f474-4368-ba8e-c0fe8e600403\ts.17a322f65e2.3ac848c2.1...ok
[22:21:07 INFO]: [AsyncWorldEdit]        * plugins\AsyncWorldEdit\undo\f654a811-f474-4368-ba8e-c0fe8e600403\ts.17a322f65e2.3ac848c2.1.idx...ok
[22:21:07 INFO]: [AsyncWorldEdit]        * plugins\AsyncWorldEdit\undo\f654a811-f474-4368-ba8e-c0fe8e600403\ts.17a323184f0.abfbf060.2...ok
[22:21:07 INFO]: [AsyncWorldEdit]        * plugins\AsyncWorldEdit\undo\f654a811-f474-4368-ba8e-c0fe8e600403\ts.17a323184f0.abfbf060.2.idx...ok
[22:21:07 INFO]: [AsyncWorldEdit]        * plugins\AsyncWorldEdit\undo\f654a811-f474-4368-ba8e-c0fe8e600403\ts.17a32321de2.4dde2c19.3...ok
[22:21:07 INFO]: [AsyncWorldEdit]        * plugins\AsyncWorldEdit\undo\f654a811-f474-4368-ba8e-c0fe8e600403\ts.17a32321de2.4dde2c19.3.idx...ok
[22:21:07 INFO]: [AsyncWorldEdit]        * plugins\AsyncWorldEdit\undo\f654a811-f474-4368-ba8e-c0fe8e600403\ts.17a32327944.952db564.3...ok
[22:21:07 INFO]: [AsyncWorldEdit]        * plugins\AsyncWorldEdit\undo\f654a811-f474-4368-ba8e-c0fe8e600403\ts.17a32327944.952db564.3.idx...ok
[22:21:07 INFO]: [AsyncWorldEdit] ...undo cleanup done.

**Comment by TNTUP** [comment]
Can confirm, having the same issue but on Paper 1.16.5 w/ WE 7.3.0-SNAPSHOT+5790-6799be

**Comment by SBPrime** [comment]
Fixed 3.9.1
